examples/next/next.config.js

```diff
@@ -12,6 +12,7 @@ function useEsbuildMinify(config, options) {
 }
 
 function useEsbuildLoader(config, options) {
+	// Specify `ts` if you're using TypeScript
```

Comment:
I don't think this is necessary. The loader we're looking for looks like this:

```js
{
	test: /\.(tsx|ts|js|mjs|jsx)$/,
	include: [
		'/esbuild-loader-examples/examples/next',
		/next[\\/]dist[\\/]next-server[\\/]lib/,
		/next[\\/]dist[\\/]client/,
		/next[\\/]dist[\\/]pages/,
		/[\\/](strip-ansi|ansi-regex)[\\/]/
	],
	exclude: [Function: exclude],
	use: [
		'/esbuild-loader-examples/node_modules/.pnpm/@next+react-refresh-utils@10.0.7_c0a78ea6e1823eafd9ea7ba010470747/node_modules/@next/react-refresh-utils/loader.js',
		[Object]
	]
}
```

So using `.js` or `.ts` will make no difference.

Reply:
I don't really know, but with '.js' it didn't work for me